src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -881,16 +881,17 @@ public void TestCapitalize()
             Assert.AreEqual(null, StandardFilters.Capitalize(null));
             Assert.AreEqual("", StandardFilters.Capitalize(""));
             Assert.AreEqual(" ", StandardFilters.Capitalize(" "));
-            Assert.AreEqual("That Is One Sentence.", StandardFilters.Capitalize("That is one sentence."));
+            Assert.AreEqual("That is one sentence.", StandardFilters.Capitalize("That Is One Sentence."));

```

Comment:
Best put such behavior change behind an opt-in flag so existing users are not surprised by the change. I filed the bug myself because the current behavior is out of spec. It has been so for a long time so it is probable some users now rely on it. Furthermore on Shopify's own forum there are customers wanting / expecting the behavior dot liquid has actually implemented, making it likely some would take a dependency on it.
Even Shopify's own wiki has contradictory statement in their wiki:
https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#standard-filters

> capitalize - capitalize words in the input sentence
Note the unfortunate plural.